components/MerkleTree/Visualisation/LinksAndNodes.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { hierarchy, Tree } from '@visx/hierarchy';
+import Nodes from './Nodes';
+import Links from './Links';
+import { Group } from '@visx/group';
+import { TreeNode } from '../Container/utils';
+
+interface ILinksAndNodes {
+    data: TreeNode;
+    sizeWidth: number;
+    sizeHeight: number;
+    origin: { x: number; y: number; };
+}
+
+export default function LinksAndNodes(props: ILinksAndNodes) {
```

Comment:
can extract props in the function params itself, just a syntactic sugar improvement.
Let's use React.FC<ITYPE> type declaration for types.